### PR TITLE
Feature/df 1660 set backlinks in ch fiches and set details

### PR DESCRIPTION
**Pull request by ganqqwerty**
the backlinks to the sets are displayed now in ch-fiche details, ch-fiche edit and ch-fiche create
image
image

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html [hunk 1]
```diff
@@ -30,22 +32,22 @@
...
     <alv-interactive-list-item *ngFor="let resolvedOccupation of resolvedOccupations; let index = index"
-                         [type]="'CH-ISCO5 ' + resolvedOccupation.occupation.chIsco5"
-                         [superTitle]="'BFS ' + resolvedOccupation.occupation.bfsCode"
-                         [multiLanguageTitle]="{
-                         de: resolvedOccupation.labelData.default,
-                         fr: resolvedOccupation.labelData.default,
-                         it: resolvedOccupation.labelData.default,
-                         en: resolvedOccupation.labelData.default
-                         }"
+                               [type]="'CH-ISCO5 ' + resolvedOccupation.occupation.chIsco5"
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html]: shouldn't CH-ISCO5 and BFS be some kind of constants here?

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
-<ng-container >
+<ng-container *ngIf="{
```
Comment (emina-str) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html]: you're using it just in one place, and just one observable, so maybe to place it just on the item in question `alv-competence-set`

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiche/ch-fiche.component.html]: well, maybe they could be placed in an i18n file, but all translations will be the same. These are just labels, they are not parameters passed to the code

File: alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-detail/competence-set-detail.component.ts [hunk 3]
```diff
@@ -23,6 +27,13 @@ export class CompetenceSetDetailComponent extends CompetenceCatalogEditorAwareCo
 
   isEdit: boolean;
 
+  backlinkCompetenceSetAction: ActionDefinition<CompetenceCatalogAction> = {
+    name: CompetenceCatalogAction.BACKLINK,
+    icon: ['fas', 'link'],
+    label: 'portal.competence-catalog.competence-sets.overview.backlink'
+  };
+
+  actions$: Observable<ActionDefinition<CompetenceCatalogAction>[]>;
```
Comment (birom) [inline alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-detail/competence-set-detail.component.ts]: What is this variable used for?

Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/competence-sets/competence-set-detail/competence-set-detail.component.ts]: not used, fixed

**Pull request merged**